client/Makefile

```diff
@@ -1,14 +1,42 @@
+CYPRESS_IMAGE?=cypress/base:12
+CYPRESS_DOCKER_ENTRYPOINT?=/bin/bash
+CYPRESS_DOCKER_CACHE=$(PWD)/.cypress-docker
+
 .env.development:
 	@echo "DIALOG_ENDPOINT=https://dev-opentutor.pal3.org/dialog/" >> .env.development
 	@echo "Set up  a default .env.development file"
 	@echo "that configures DIALOG_ENDPOINT=https://dev-opentutor.pal3.org/dialog/."
 	@echo "This file should not be committed. "
 	@echo "Feel free to change to point to a local server if you're running one."
 
+.PHONY: clean-cypress-snapshot-cache
+clean-cypress-snapshot-cache:
+	rm -rf $(CYPRESS_DOCKER_CACHE)
+
 .PHONY: clean
-clean:
+clean: clean-cypress-snapshot-cache
 	rm -rf .cache public node_modules
 
+.PHONY: cypress-docker
+cypress-docker:
+	mkdir -p $(CYPRESS_DOCKER_CACHE)/.npm
```

Comment:
looks like I had removed the volume mounts in `docker run` for `.npm` and `.root.cache` (because they actually made the process slower), so can remove the lines that create those two folders